prelude/toolchains/conan/conan_init.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import shutil
+import subprocess
+
+import conan_common
+
+
+def conan_profile(
+        conan,
+        user_home,
+        trace_log):
+    env = conan_common.conan_env(
+            user_home=user_home,
+            trace_log=trace_log)
+
+    # TODO[AH] Allow users to define additional remotes.
```

Comment:
A conan_remote rule would be cool. You could pass it in similar to the profile similar to deps.

Reply:
Yes, a `conan_remote` rule sounds like a good way to do this.